aspnetcore/migration/mvc.md

```diff
@@ -194,26 +192,9 @@ For information about how to configure bundling and minification, see [Bundling
 
 ## Solving HTTP 500 errors
 
-There are many problems that can cause a HTTP 500 error message that contain no information on the source of the problem. For example, if the *Views/_ViewImports.cshtml* file contains a namespace that doesn't exist in your project, you'll get a HTTP 500 error. To get a detailed error message, add the following code:
-
-```csharp
-public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
-{
-    if (env.IsDevelopment())
-    {
-         app.UseDeveloperExceptionPage();
-    }
-
-    app.UseStaticFiles();
-
-    app.UseMvc(routes =>
-    {
-        routes.MapRoute(
-            name: "default",
-            template: "{controller=Home}/{action=Index}/{id?}");
-    });
-}
-```
+There are many problems that can cause a HTTP 500 error message that contain no information on the source of the problem. For example, if the *Views/_ViewImports.cshtml* file contains a namespace that doesn't exist in your project, you'll get a HTTP 500 error. By default in ASP.NET Core Applications, the `UseDeveloperExceptionPage` extension is added to the `IApplicationBuilder` and executed when the configuration is *Development*. This is detailed in the following code:
```

Comment:
> There are many problems that can cause a HTTP 500 error message that contain no information on the source of the problem

ASP.NET Core will turn unhandled exceptions in your web app to HTTP 500 error responses. 

Normally, error details are not included in these responses to prevent disclosure of potentially sensitive information about the server.

Reply:
adding additional detail